app/helpers/versions_helper.rb

```diff
@@ -162,5 +162,4 @@ def user_change_description(version)
   def users_role_change_description(version)
     version.event == 'create' ? 'added' : 'removed'
   end
-  
-end
+end
```

Comment:
Please add empty line at the end of the file, for styling purposes.
You can probably configure your text editor to do that automatically for you.

Reply:
Already did :)